src/core/Form/Combobox/ChipList/ChipList.tsx

```diff
@@ -0,0 +1,46 @@
+import React, { Component } from 'react';
+import { default as styled } from 'styled-components';
+import classnames from 'classnames';
+import { HtmlDiv, HtmlDivProps } from '../../../../reset';
+import { AutoId } from '../../../../utils/AutoId';
+import { baseStyles } from './ChipList.baseStyles';
+
+const baseClassName = 'fi-chip-list';
+
+const chipListClassNames = {
+  wrapper: `${baseClassName}_wrapper`,
```

Comment:
This is really minor, but could this be `content_wrapper`? Now it's a bit misleading as there is first the actual chip list and then the wrapper inside it

Reply:
This is now changed to be like suggested, `content_wrapper`.